web3/utils/encoding.py

```diff
@@ -231,16 +229,10 @@ def text_if_str(to_type, text_or_primitive):
 
     @param to_type is a function that takes the arguments (primitive, hexstr=hexstr, text=text),
         eg~ to_bytes, to_text, to_hex, to_int, etc
-    @param hexstr_or_primitive in bytes, str, or int. (or unicode in py2)
-        In Python 2, only a unicode object will be interpreted as unicode text
-        In Python 3, only a str object will be interpreted as interpreted as unicode text
+    @param hexstr_or_primitive in bytes, str, or int.
+        In Python 3, only a str object will be interpreted as unicode text
```

Comment:
It is all Python 3 now.

Reply:
I guess its safe to delete all the "python 3" comments?

Reply:
:+1: